README.md

```diff
@@ -27,6 +27,20 @@ At each Element release, the SDK module is copied to a dedicated repository: htt
 The version 1.0.0 of Element still misses some features which was previously included in Riot-Android.
 The team will work to add them on a regular basis.
 
+# Releases to app stores
+
+There is some delay between when a release is created and when it appears in the app stores (Google Play Store and F-Droid). Here are some of the reasons:
+
+* Not all versioned releases that appear on GitHub are considered stable. Each release is first considered beta: this continues for at least two days. If the release is stable (no serious issues or crashes are reported), then it is released as a production release in Google Play Store, and a request is sent to F-Droid too.
+* Each release on the Google Play Store undergoes review by Google before it comes out. This can take an unpredictable amount of time. In some cases it has taken several weeks.
+* In order for F-Droid to guarantee that the app you receive exactly matches the public source code, they build releases themselves. When a release is considered stable, Element staff inform the F-Droid maintainers and it is added to the build queue. Depending on the load on F-Droid's infrastructure, it can take some time for releases to be built. This always takes at least 24 hours, and can take several days.
+
+If you would like to receive releases more quickly (bearing in mind that they may not be stable) you have a number of options:
+
+1. [Sign up to receive beta releases](https://play.google.com/apps/testing/im.vector.app) via the Google Play Store.
+2. Install a [release APK](https://github.com/vector-im/element-android/releases) directly - download the relevant .apk file and allow installing from untrusted sources in your device settings.
```

Comment:
could it be worth to precise that GH builds are only ment for GPlay variant? Or is this a wrong assumption?

Thanks for this anyway, really helpful :)

Reply:
Ah yes definitely.  @bmarty is that correct?

Reply:
No, F-Droid debug is also build by GitHub action as per https://github.com/vector-im/element-android/blob/develop/.github/workflows/build.yml#L22. I do not know where are exported the APKs though

Reply:
Yes, so far release versions of F-Droid variant are not build by us, just debug versions.

Reply:
(I should have read more carefully the text)

Reply:
@Ornanovitch I made a new PR with a small update to mention what you suggested, thanks!